liquid.py

```diff
@@ -117,6 +135,10 @@ def get(self, env_key, ini_path, default=None):
 
         return default
 
+    def validate_collection_name(self, name):
```

Comment:
This method never uses the `Configuration` object, so it should be a `@classmethod` or a global function.

Also, I think it's simpler to have a `Collection.is_name_valid` classmethod that just checks if the name is correct or not. The caller (in this case, the `initcollection` command) will throw errors and print messages.

Reply:
The caller is not the `initcollection` command. Also, this method is not supposed to be used outside Configuration, that's why I didn't bother making it a class method.